test/test_ads_utils.py

```diff
@@ -102,6 +114,26 @@ def test_get_existing_models_list_returns_known_models(self):
         for downloaded_model in get_existing_models_list():
             assert downloaded_model in known_models
 
+    @pytest.mark.unit
+    def test_get_existing_models_list_returns_none_for_no_existing_models(self):
+        # Move content inside model folder to test/__test_files__/tmp_folder, without moving the folder itself
+        if not (self.tmp_folder / "models").exists():
+            (self.tmp_folder / "models").mkdir()
+
+        for file in self.model_dir.iterdir():
+            print(file)
+            # Make temp dir
+            shutil.move(file, self.tmp_folder / "models")
+        
+        # Main test
+        assert get_existing_models_list() == None
```

Comment:
Assuming the suggested change is made, then maybe this test would be:

```python
models_list = get_existing_models_list()
assert isinstance(models_list, list) and len(models_list) == 0
```

Reply:
I Iike it - will make the change and push!